package/MDAnalysis/coordinates/TRC.py

```diff
@@ -0,0 +1,399 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+
+"""GROMOS11 trajectory reader --- :mod:`MDAnalysis.coordinates.TRC`
+====================================================================
+
+Reads coordinates, timesteps and box-sizes from GROMOS11 TRC trajectories.
+
+To load the trajectory into :class:`~MDAnalysis.core.universe.Universe`,
+you need to provide topology information using a pdb::
+
+    import MDAnalysis as mda
+    u = mda.Universe("topology.pdb", ["md_1.trc.gz","md_2.trc.gz"],
+    continuous=True)
```

Comment:
Let's ignore the line limitations to make it look nicer
```suggestion
    u = mda.Universe("topology.pdb", ["md_1.trc.gz","md_2.trc.gz"], continuous=True)
```

Reply:
For the formatting, I will stick to the output of `black` as recommended above.
The standard line length with `black` seems to be 88 instead of flake8's 79 limit.

Nevertheless, the contributing guide of mdanalysis states a strict 79 char limit. If should enforce `black` with 79 chars, please tell me.

Reply:
I *think* 88 chars will still be ok. Our own .flake8 file specifies 88 chars. The contributor guide appears to be outdated. I raised https://github.com/MDAnalysis/UserGuide/issues/347

Have a look at the output of the darker CI, eg https://github.com/MDAnalysis/mdanalysis/actions/runs/6890070863/job/18742353249?pr=4292#step:4:533

It's advisory (so it normally does not turn red) but you can see under *darker-main-code* a number of warnings/errors — if these originate from your code, please address.